src/Diagnostics.ModelsAndUtils/Models/DataTableResponseObject.cs

```diff
@@ -17,7 +17,7 @@ public class DataTableResponseObject
 
         public IEnumerable<DataTableResponseColumn> Columns { get; set; }
 
-        public string[][] Rows { get; set; }
+        public dynamic[,] Rows { get; set; }
```

Comment:
breaking change?

Reply:
no it should not be. Everything was working end to end. In JSON the two would be the same. This object is not accessed by the detector code, as we give them the data table and then convert it to this.